src/screens/FeedScreen/Content.js

```diff
@@ -246,8 +247,8 @@ const Content = ({
 
   return (
     <Pressable
-      onLayout={hanldeHeightContainer}
-      onPress={onPress}
+      onLayout={handleHeightContainer}
+      onPress={item?.isBlurredPost ? null : () => onPress()}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional check for `item?.isBlurredPost` before calling `onPress()` could potentially lead to confusion as it silently fails without any feedback to the user. Consider providing some form of feedback or disabling the pressable component altogether when `item.isBlurredPost` is true.
```diff
- onPress={item?.isBlurredPost ? null : () => onPress()}
+ onPress={item?.isBlurredPost ? () => alert('This post is blurred.') : () => onPress()}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->